examples/MinimalMqttClient/MinimalMqttClient.ino

```diff
@@ -0,0 +1,45 @@
+/*
+  BasicMQTTClient.ino
+  This shows the basic handling of MQTT and Wifi connection.
+  Once it connects successfully to a Wifi network and a MQTT broker,
+  it subscribes to a topic and send a message to it.
+*/
+
+#include "EspMQTTClient.h"
+
+// Change these to customize where data will go
+String topic = "topicname";
+char userID[] = "yourname";
```

Comment:
Here is `String` and `char[]` used.

Personally I like to use the `#define` for Texts as you can concat multiple ones at compile time then (`"foo" "bar"` will be `"foobar"`). But I also understand why the `#define` is not used when explaining C++.

It should be the same to be easier to get for the user as its both just some text.